sopel_remind/backend.py

```diff
@@ -136,24 +145,78 @@ def parse_in_delta(line: str) -> Tuple[timedelta, str]:
     return delta, message
 
 
-def parse_at_time(line: str) -> Tuple[time, str]:
+def parse_at_time(
+    line: str,
+    today: datetime,
+    user_tz: pytz.BaseTzInfo,
```

Comment:
If the parameter currently called `today` must be an Aware datetime, also passing a `user_tz` parameter is redundant, isn't it? `today` already knows what timezone it's in (accessible via `today.tzinfo.zone`).

Reply:
Hm... it might be an oversight from me, yes.